microshift_getting-started/microshift-architecture.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="microshift-architecture"]
+= Architecture
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-architecture
+
+toc::[]
+
+== The benefits of running containers on the edge
+
+{product-title} is a single-node container orchestration runtime designed for running applications in low-resource edge environments. Like {OCP}, {product-title} is based on Kubernetes and retains many of the benefits of using containers, bringing those benefits to the edge. Because edge and small-form-factor computing are by nature created with only the essentials needed to operate, so is {product-title}.
```

Comment:
```suggestion
{product-title} is a single-node container orchestration runtime designed for running applications in low-resource edge environments. Like {OCP}, {product-title} is based on Kubernetes and retains many of the benefits of using containers, bringing those benefits to the edge. {product-title} contains only the essentials needed to operate in edge and small form factor computing environments.
```
Rewording suggestion to avoid passive voice.